framework/src/main/java/org/checkerframework/common/subtyping/SubtypingChecker.java

```diff
@@ -23,20 +24,37 @@
  */
 @SupportedOptions({"quals", "qualDirs"})
 public final class SubtypingChecker extends BaseTypeChecker {
-    @Override
-    public Collection<String> getSuppressWarningsKeys() {
+
+    /**
+     * Compute getSuppressWarningsKeys, based on the names of all the qualifiers.
+     *
+     * <p>Provided for the convenience of checkers that do not subclass Subtyping Checker (because
+     * it is final). Clients should call it like:
+     *
+     * <pre>{@code
+     * SubtypingChecker.getSuppressWarningsKeys(this.visitor, super.getSuppressWarningsKeys());
+     * }</pre>
+     *
+     * @param visitor the visitor
+     * @param superSupportedTypeQualifiers the result of super.getSuppressWarningsKeys(), as
+     *     executed by checker
+     */
+    public static Collection<String> getSuppressWarningsKeys(
```

Comment:
This change should be added to the description of this pull request or moved to a separate request.